<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="anujs-MacBook-Pro.local" ignored="1" name="Default test(failed)" tests="2" failures="1" timestamp="2020-04-18T21:08:18 EDT" time="0.041" errors="0">
  <testcase name="test2" time="0.007" classname="demo.TestNGRetryDemo"/>
  <testcase name="test3" time="0.002" classname="demo.TestNGRetryDemo">
    <failure type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: 
at demo.TestNGRetryDemo.test3(TestNGRetryDemo.java:29)
at java.util.ArrayList.forEach(ArrayList.java:1257)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- test3 -->
  <testcase name="test1" time="0.0" classname="demo.TestNGRetryDemo">
    <ignored/>
  </testcase> <!-- test1 -->
</testsuite> <!-- Default test(failed) -->
